cases/toga/case.yml

```diff
@@ -0,0 +1,26 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:indentation
+# yamllint disable rule:comments-indentation
+# too many spelling things, spell-checker: disable
+---
+- case: toga-latest
+  # Integer version, increased when results become invalidated by cases changes.
+  version: 1
+  filename: "TogaCase.py"
```

Comment:
I prefer names like "*Using.py"  for some reason.

Reply:
preference, or requirement?